content/en/docs/external-add-ons/feature-store/OWNERS

```diff
@@ -1,3 +0,0 @@
-approvers:
-  - woop
-  - franciscojavierarceo
```

Comment:
Can we keep this one?

Reply:
The purpose of these sections is not to provide up to date information, it is to provide a brief "evergreen" explanation of what the add-on is, and how it relates to Kubeflow.

I would prefer if the root approvers of the Kubeflow website are aware and approve each proposed change, to avoid messaging conflicts.

Even without being an approver, you or others can raise PRs to propose updates.